README.md

```diff
@@ -6,25 +6,25 @@
 [![Coverage Status](https://coveralls.io/repos/freedomofpress/securedrop/badge.svg?branch=develop&service=github)](https://coveralls.io/github/freedomofpress/securedrop?branch=develop)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/freedomofpress/securedrop?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-SecureDrop is an open-source whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. It was originally created by the late Aaron Swartz and is currently managed by [Freedom of the Press Foundation](https://freedom.press).
+SecureDrop is an open-source whistleblower submission system that media organizations can use to securely accept documents from, and communicate with anonymous sources. It was originally created by the late Aaron Swartz and is currently managed by the [Freedom of the Press Foundation](https://freedom.press).
 
-SecureDrop's documentation is now built and hosted by [Read the Docs](https://readthedocs.org): https://docs.securedrop.org. If you are still trying to use links to Markdown files on our Github to read documentation, please update your bookmarks.
+SecureDrop documentation is now built and hosted by [Read the Docs](https://readthedocs.org): https://docs.securedrop.org. If you are still trying to use links to Markdown files on our GitHub to read documentation, please update your bookmarks.
 
 ## Found an issue?
 
-If you're here because you want to report an issue in SecureDrop, please observe the following protocol to report an issue responsibly:
+If you're here because you want to report an issue in SecureDrop, please observe the following protocol to do so responsibly:
 
 * If you want to report a **_security issue_**, please use our [bug bounty hosted by Bugcrowd](https://bugcrowd.com/freedomofpress).
-* If the issue does not have a security impact, just create a [Github Issue](https://github.com/freedomofpress/securedrop/issues/new).
+* If filing the issue does not impact security, just create a [GitHub Issue](https://github.com/freedomofpress/securedrop/issues/new).
 
 ## How to Install SecureDrop
 
 See the [Installation Guide](https://docs.securedrop.org/en/latest/#installtoc).
 
 ## How to Use SecureDrop
 
-* [How to use SecureDrop as a source](https://docs.securedrop.org/en/latest/source.html)
-* [How to use SecureDrop as a journalist](https://docs.securedrop.org/en/latest/journalist.html)
+* [As a source](https://docs.securedrop.org/en/latest/source.html)
+* [As a journalist](https://docs.securedrop.org/en/latest/journalist.html)
```

Comment:
I like this change, but you should link to the `stable` documentation (which corresponds to the latest stable release of SecureDrop running on production instances and is therefore appropriate for end users), rather than the `latest` documentation (which is built from the latest commits on the development branch and is therefore appropriate only for developers).